rasa/utils/tensorflow/temp_keras_modules.py

```diff
@@ -175,12 +175,13 @@ def fit(
             data_handler._initial_epoch = self._maybe_load_initial_epoch_from_ckpt(  # pylint: disable=protected-access # noqa: E501
                 initial_epoch
             )
+            logs = None
```

Comment:
Do we need this and line 198. What do they do that line 195 doesn't

Reply:
This  is a change tensorflow introduced in the code that we copied:
https://github.com/tensorflow/tensorflow/blob/r2.6/tensorflow/python/keras/engine/training.py#L1180

I imagine the first one allows for handling the case where `data_handler.enumerate_epochs()` == 0 more gracefully, and the second one prevents problems if the log output is for some reason malformed by running it through `sync_to_numpy_or_python_type`. Granted, it does look pretty weird.

Reply:
Just a side note: We should try to resume activity on [this](https://github.com/tensorflow/tensorflow/pull/45338), so that we don't have to maintain `tmp_keras_module.py`

Reply:
> try to resume activity on this

That would make me so happy 😭 Not sure how to push for this. I tried to make an issue for [the other one](https://github.com/RasaHQ/rasa/issues/9232) and it was promptly deprioritised. Maybe I should make an issue for this one and raise them both for Engine? wdyt?